packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security-rbac.cy.ts

```diff
@@ -33,12 +35,41 @@ describe("TESTS: Access secured by Elytron OIDC client with RBAC", () => {
     cy.get("#username").type("userwithrole");
     cy.get("#password").type("password");
     cy.get("#kc-login").click();
+    cy.get("#tlc-access-control").should("have.text", "Access Control");
     cy.verifyUserName("userwithrole");
     cy.verifyUserRole("Administrator");
     cy.logoutFromWebConsole();
     verifyNotLoggedIn(keycloak);
   });
 
+  it("Logs in successfully and logs out by user with letter case insensiteve role", () => {
+    cy.visit(`/?connect=${wildfly}#home`);
```

Comment:
>    cy.visit(`/?connect=${wildfly}#home`);
>    cy.get("#username").type("userwithrole");
>    cy.get("#password").type("password");
>    cy.get("#kc-login").click();

As those four lines are in every test method, can we introduce a new local method `logIn(username: String, password: String)`instead?

Reply:
new local method introduced.